chat-widget/src/firstresponselatency/Constants.ts

```diff
@@ -0,0 +1,34 @@
+export enum ScenarioType {
+    UserSendMessageStrategy = "UserSendMessageStrategy",
+    SystemMessageStrategy = "SystemMessageStrategy",
+    ReceivedMessageStrategy = "ReceivedMessageStrategy"
+}
+
+export type MessagePayload = {
```

Comment:
`MessagePayload` is a generic name. It may be reused in other features in the future. Should a prefix be added to be more distinguishable? `FRLMessagePayload`?

Reply:
MessagePayload can be extended, I'd like to keep the name since it shares type with ACS incoming messages and FRL